shared/actions/kbfs/index.platform.desktop.js

```diff
@@ -163,6 +164,62 @@ function* installFuseSaga(): SagaGenerator<any, any> {
   yield put(finishedAction)
 }
 
+function installCachedDokan(): Promise<*> {
+  return new Promise((resolve, reject) => {
+    const regedit = require('regedit')
+    regedit.list('HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall', (err, programKeys) => {
+      if (err) {
+        reject(err)
+      } else {
+        var programKeyNames =
+          programKeys['HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall'].keys
+
+        for (var keyName of programKeyNames) {
+          var programKey = 'HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\' + keyName
+
+          regedit.list(programKey, (err, program) => {
+            if (err) {
+              reject(err)
+            } else {
+              for (var p in program) {
+                var vals = program[p]
+                var displayName, publisher
+                var modifyPath = ''
+                for (var v in vals) {
+                  if (vals[v]['DisplayName']) {
+                    displayName = vals[v]['DisplayName'].value
+                  }
+                  if (vals[v]['Publisher']) {
+                    publisher = vals[v]['Publisher'].value
+                  }
+                  if (vals[v]['ModifyPath']) {
+                    modifyPath = vals[v]['ModifyPath'].value
+                  }
+                }
+                if (displayName === 'Keybase' && publisher === 'Keybase, Inc.') {
+                  modifyPath = modifyPath.replace(/"/g, '')
+                  modifyPath = modifyPath.replace(' /modify', '')
+                  console.log(modifyPath)
+                  execFile(modifyPath, [
+                    '/modify',
+                    'driver=1',
+                    'modifyprompt=Click "Repair" to view files in Explorer',
```

Comment:
Can the GUI call into the service to do the uninstall to avoid having to duplicate this code/logic? Perhaps in a future PR?